### PR TITLE
Clone parking checklist

### PR DESCRIPTION
**Pull request by dneise**
So .. I've sometimes gotten developer calls in the night, or also during the day regarding a failed "ParkingChecklistFilled" check.

I looked into the code and saw, that this table is actually not cloned. So I thought, let's go ahead and clone all tables the SH is reading from .. not just some of them.

The reason why this table is not cloned is, that is does not reside in the same DB as the other tables. It is in the sandbox DB.

### CODE REVIEW

**Comment by dneise** [comment]
At this point in time, I realize, that due to the cloning time delay of 5 minutes, the SH will always have an image of the Database which is max. 5 minutes too old. So this check:
https://github.com/fact-project/shifthelper/blob/6b841be6f28fb4cac2cb56b6758cfbfd06c55a8b/shifthelper/__main__.py#L226-L230

might fire when it is 10min after shutdown, but the parker has already filled the checklist, but the cloner has not yet cloned the DB again, so the SH looks simply at old data.

**Comment by dneise** [comment]
Now the condition "is_last_shutdown_already_10min_past" takes the cloning time into account.

**Comment by dneise** [comment]
The next point I see is:
The SH usually makes a connection to the "cloned_db". but the default here: 
https://github.com/fact-project/shifthelper/blob/6b841be6f28fb4cac2cb56b6758cfbfd06c55a8b/shifthelper/tools/__init__.py#L44-L47
Is the live DB on LaPalma.

I think it would be good to make this default somehow reflect, that we usually use the cloned DB.

The only part in the code not using the cloned DB, is the DB_cloner .. which makes sense.

**Comment by dneise** [comment]
This PR got a bit too big already ... sorry.

**Comment by dneise** [comment]
Maybe we could think about merging this?

File: shifthelper/conditions.py [hunk 1]
```diff
@@ -345,7 +350,10 @@ def is_nobody_on_shift():
 @log_call_and_result
 def is_last_shutdown_already_10min_past():
     '''Last Shutdown is already 10min past'''
-    return get_last_shutdown() + timedelta(minutes=10) < datetime.utcnow()
+    return (
+        get_last_shutdown() + timedelta(minutes=10) + TIME_BETWEEN_CLONES <
```
Comment (MaxNoe) [inline shifthelper/conditions.py]: I'm really not sure about this one. Maybe drop this and just clone the parking checklist more often? Like once a minute?

Comment (dneise) [reply shifthelper/conditions.py]: I wonder if cloning more often defeats the purpose of cloning...

Comment (dneise) [reply shifthelper/conditions.py]: but maybe your are right

**Comment by dneise** [comment]
ok merge?

**Pull request merged**